src/css/grid.styl

```diff
@@ -111,6 +111,15 @@ grid-maker($width = false, $gutter = false, $size = false, $cols = 12)
     .container
       width: $width
 
+    .container-large
+      width: 100%
+
+    .container-medium
+      width: $width*0.80
+
+    .container-small
+      width: $width*0.70
+
```

Comment:
actually scratch that. use grid items & offsets within the normal container rather than various container sizes